### ISSUE TITLE
Compilation fails on Mac due to no OpenMP support

### ISSUE THREAD

**Issue by nippoo** [klusta-team/klustakwik2#50]
```
26 warnings generated.
ld: library not found for -lgomp
clang: error: linker command failed with exit code 1 (use -v to see invocation)
E
======================================================================
ERROR: Failure: ImportError (Building module klustakwik2.numerics.cylib.e_step_cy failed: ["LinkError: command 'gcc' failed with exit status 1\n"])
----------------------------------------------------------------------
```

**Comment by thesamovar** [comment]
This should be relatively easily solvable by disabling OpenMP when building on Mac. Maybe some complexities to do with having prange or range in the Cython code. I won't do anything on this for the moment because we might drop Cython code if the idea of distributing over machines/processes works out.

**Comment by nippoo** [comment]
Coolio. Just makes it a bit trickier to develop on my MacBook, haha!

**Comment by thesamovar** [comment]
Of course if you want to fix it I'd be happy with that. :)

**Comment by nippoo** [comment]
This is now closed thanks to #52
